mlir/include/mlir/Dialect/Arith/IR/Arith.h

```diff
@@ -78,6 +84,8 @@ class ConstantFloatOp : public arith::ConstantOp {
   using arith::ConstantOp::ConstantOp;
   static ::mlir::TypeID resolveTypeID() { return TypeID::get<ConstantOp>(); }
 
+  using arith::ConstantOp::build;
+
```

Comment:
Please don't make changes that are irrelevant to the PR description. It says `ConstantIntOp`, it shouldn't touch `ConstantFloatOp` or say otherwise.